src/app/(main)/form/[id]/submissions-table.tsx

```diff
@@ -260,6 +267,10 @@ export function SubmissionsTable({
                 <TableRow
                   key={row.id}
                   data-state={row.getIsSelected() && "selected"}
+                  className={cn(
+                    row.original.isSpam &&
+                      "bg-red-100 duration-300 hover:bg-red-50",
```

Comment:
oh, it's the on hover classes that got me here. I was thinking a new button was added